### PR TITLE
[16.0][MIG] account_statement_import_online_ofx: Migration to 16.0

### PR DESCRIPTION
**Pull request by PauBForgeFlow**

### CODE REVIEW

**Review by cormaza — APPROVED** [review]
LGTM +

**Review by bofiltd — APPROVED** [review]
LGTM

File: account_statement_import_online_ofx/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+{
+    "name": "Online Bank Statements: OFX",
+    "version": "16.0.1.0.0",
+    "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/bank-statement-import",
+    "license": "AGPL-3",
+    "category": "Accounting",
+    "summary": "Online bank statements for OFX",
+    "depends": ["account_statement_import_online", "hr"],
```
Comment (StefanRijnhart) [inline account_statement_import_online_ofx/__manifest__.py]: Can you clarify why the new dependency on `hr` is needed?

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
The dependency on `hr` is something non desired.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**